Tests/XCUITests/FxScreenGraph.swift

```diff
@@ -980,6 +985,12 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.backAction = cancelBackAction
     }
 
+    map.addScreenState(TabsSettings) { screenState in
```

Comment:
Here's the `TabSettings` page. The `ToggleInactiveTabs` is not used in "Jump Back In" section but is included for completeness of the page.